plugins/services/src/js/routes/services.js

```diff
@@ -25,191 +24,194 @@ function buildServiceCrumbs(router) {
-                              {
-                                label: 'Volumes',
-                                route: {
-                                  params: router.getCurrentParams(),
-                                  to: '/services/overview/:id/tasks/:taskID/volumes'
+                    children: [
+                      {
+                        type: IndexRoute,
+                        component: TaskDetailsTab,
+                        isTab: true,
+                        title:'Details',
+                        buildBreadCrumb() {
+                          return {
+                            parentCrumb: '/services/overview/:id/tasks/:taskID',
+                            getCrumbs() { return []; }
+                          };
+                        }
+                      },
+                      {
+                        type: Route,
+                        path: 'files',
+                        title:'Files',
+                        component: TaskFilesTab,
+                        isTab: true,
+                        hideHeaderNavigation: true,
+                        fileViewerRoutePath: '/services/overview/:id/tasks/:taskID/view/?:filePath?/?:innerPath?',
+                        buildBreadCrumb() {
+                          return {
+                            parentCrumb: '/services/overview/:id/tasks/:taskID',
+                            getCrumbs() { return []; }
+                          };
+                        }
+                      },
+                      {
+                        type: Route,
+                        dontScroll: true,
+                        component: TaskFileViewer,
+                        title:'Logs',
+                        path: 'view/?:filePath?/?:innerPath?',
+                        isTab: true,
+                        buildBreadCrumb() {
+                          return {
+                            parentCrumb: '/services/overview/:id/tasks/:taskID',
+                            getCrumbs() { return []; }
+                          };
+                        }
+                      },
+                      {
+                        type: Route,
+                        path: 'volumes',
+                        component: VolumeTable,
+                        isTab: true,
+                        title:'Volumes',
+                        buildBreadCrumb() {
+                          return {
+                            parentCrumb: '/services/overview/:id/tasks/:taskID',
+                            getCrumbs() { return []; }
+                          };
+                        }
+                      },
+                      {
+                        type: Route,
+                        path: 'volumes/:volumeID',
+                        component: VolumeDetail,
+                        buildBreadCrumb() {
+                          return {
+                            parentCrumb: '/services/overview/:id/tasks/:taskID/volumes',
```

Comment:
We talked about this and we came to the conclusion that this is better readable.